docs/source/1.0/spec/aws/aws-core.rst

```diff
@@ -1525,7 +1524,7 @@ HTTP header. Thus, the resolved location is always ``Header``.
 
 .. _Header-based auth: https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-header-based-auth.html
 .. _Streaming-signing auth: https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-streaming.html
-.. _chunked trailer part: https://tools.ietf.org/html/rfc7230#section-4.1.2
+.. _chunked trailer part: https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-streaming.html
```

Comment:
should this be a separate commit (I'm fine without a separate PR)

Reply:
Probably should have been, but not worth a ton of effort to correct either.